server/db/migrations/20201026031032-add-kit_version-to-status.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const TABLE = 'Statuses';
+const COLUMN = 'kit_version';
+
+module.exports = {
+	up: (queryInterface, Sequelize) =>
+		queryInterface.addColumn(TABLE, COLUMN, {
+			type: Sequelize.STRING,
+			defaultValue: '2.0.0'
+		}),
+	down: (queryInterface, Sequelize) =>
+		queryInterface.removeColumn(TABLE, COLUMN)
+};
```

Comment:
kit_version migration file

Reply:
dont add a default value for version. just leave it as empty @brandonkimmmm

Reply:
@abeikverdi  i've removed the default value in seeder and migration and made `allowNull` true in the model. `setKitVersion` will set `kit_version` to `process.env.KIT_VERSION || status.kit_version`